# disable umount -r for non-root users to fix privilege escalation (CAN-2005-2876)

--- util-linux-2.12a/mount/umount.c.can-2005-2876	2005-09-19 20:18:32.011686865 -0600
+++ util-linux-2.12a/mount/umount.c	2005-09-19 20:19:02.699173130 -0600
@@ -695,7 +695,7 @@
 
 	if (getuid () != geteuid ()) {
 		suid = 1;
-		if (all || types || nomtab || force)
+		if (all || types || nomtab || force || remount)
 			die (2, _("umount: only root can do that"));
 	}
 
